.github/labeler.yml

```
@@ -0,0 +1,89 @@
+"Category: Admin":
+  - redbot/cogs/admin/*
+"Category: Alias":
+  - redbot/cogs/alias/*
+"Category: Audio API":
+  - redbot/cogs/audio/apis/*
+  - redbot/cogs/audio/data/*
+"Category: Audio Cog":
+  - redbot/cogs/audio/*
+  - redbot/cogs/audio/core/*
```

Audio API is not yet a thing, so there's no actual folder related to it - everything in `redbot/cogs/audio` (except `locales` folder) should have the `Audio Cog` label applied
```suggestion
"Category: Audio Cog":
  - redbot/cogs/audio/**/*
  - !redbot/cogs/audio/**/locales/*
```